frontend/packages/ux-editor/src/components/config/componentSpecificContent/Summary2/Summary2ComponentReferenceSelector.tsx

```diff
@@ -9,21 +9,27 @@ type Summary2ComponentTargetIdProps = {
   onValueChange: (value: string) => void;
 };
 
-export const Summmary2ComponentTargetSelector = ({
+export const Summmary2ComponentReferenceSelector = ({
   label,
   value,
   options,
   onValueChange,
 }: Summary2ComponentTargetIdProps) => {
   const invalidOption = Boolean(value) && !options.some((option) => option.id === value);
   const { t } = useTranslation();
+
+  const errorMessage = invalidOption
+    ? t('ux_editor.component_properties.target_invalid')
+    : false || !value
+      ? t('ux_editor.component_properties.enum_Required')
+      : false;
```

Comment:
I wonder if we can simlify this to somthign like this:?🙂
```suggestion
const invalidMessage = invalidOption && t('ux_editor.component_properties.target_invalid');
  const requiredMessage = !value && t('ux_editor.component_properties.enum_Required');
  const errorMessage = invalidMessage || requiredMessage || false;
```

Reply:
Very nice! A lot more readable and clear👍